toolchain/check/check.cpp

```diff
@@ -656,6 +657,43 @@ auto NodeIdTraversal::Next() -> std::optional<Parse::NodeId> {
   }
 }
 
+// Emits a diagnostic for each declaration in context.definitions_required()
+// that doesn't have a definition.
+static auto DiagnoseMissingDefinitions(Context& context,
+                                       Context::DiagnosticEmitter& emitter)
+    -> void {
+  CARBON_DIAGNOSTIC(MissingDefinitionInImpl, Error,
+                    "No definition found for declaration in impl file");
+  for (SemIR::InstId decl_inst : context.definitions_required()) {
```

Comment:
We typically suffix ID variables with "_id", can you please do similar here?
```suggestion
  for (SemIR::InstId decl_inst_id : context.definitions_required()) {
```